tests/cypress/start_cypress_tests.sh

```diff
@@ -73,6 +77,7 @@ echo -e "\tCYPRESS_STANDALONE_TESTSUITE_EXECUTION: $CYPRESS_STANDALONE_TESTSUITE
 echo -e "\tCYPRESS_coverage       : $CYPRESS_coverage"
 echo -e "\tCYPRESS_TAGS_INCLUDE          : $CYPRESS_TAGS_INCLUDE"
 echo -e "\tCYPRESS_TAGS_EXCLUDE          : $CYPRESS_TAGS_EXCLUDE"
+echo -e "\tCYPRESS_API_SERVER_URL : $CYPRESS_API_SERVER_URL"
```

Comment:
this is same as `CYPRESS_OPTIONS_HUB_CLUSTER_URL`

Reply:
Updated.